atm.js

```diff
@@ -1 +1,27 @@
-// your code goes here!
+var deposit1 = document.querySelector ("#checkingDepositButton")
+var withdraw1 = document.querySelector ("#checkingWithdrawalButton")
+var deposit2 = document.querySelector ("#savingsDepositButton")
+var withdraw2 = document.querySelector ("#savingsWithdrawalButton")
+
+function handleClickEvent (){
+
+  console.log("I was clicked");
+  console.log(this);
+
+}
+// This is the eventListener.
+button.addEventListener("click", handleClickEvent );
+
+
+
+//This is the eventListener.
+deposit1.addEventListener("click", handleClickEvent );
+withdraw1.addEventListener("click", handleClickEvent );
+deposit2.addEventListener("click", handleClickEvent );
+withdraw2.addEventListener("click", handleClickEvent );
+input.addEventListener("onClick" = function());
+
+document.getElementById("#checkingDepositButton").addEventListener("click", something);
```

Comment:
It's fine to have psuedo-codey stuff like this in your script when you are still working on something, just make sure to comment it out so it doesn't cause the rest of your script to break.